src/index.ts

```diff
@@ -34,6 +35,8 @@ export async function execute(filenames: string[], options: Partial<ExecutionOpt
 
     let manifest = await PP.getManifest(executionOptions.root);
 
+    let nodeDefs = await getComponentDefinitionMap(executionOptions.root);
```

Comment:
should this be optional in case people want to run `brs` on source files only and don't have any components to parse or don't want to parse components?

Reply:
In the case of no components I believe glob should return empty.  We can explore adding an option in the future to exclude component parsing.